build/charts/antrea/templates/simulator/statefulset.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       serviceAccountName: antrea-agent
       initContainers:
       - name: init-inotify-limit
-        image: projects.registry.vmware.com/antrea/busybox:latest
+        image: antrea/busybox:latest
```

Comment:
Unrelated, but I'd rather have use use the `ubuntu:22.04` image for this

Reply:
Updated.